### PR TITLE
Fix: Repair conflicts and emove the `Debug` ignore errors.

### PR DESCRIPTION
**Pull request by kai890707**
**Description** 
See #7731

Sorry, There was a conflict with a previous modification (#7796) and due to my operational issues I have reorganized the complete modification into a new PR.
 
 **Checklist:**
 
 * [x]  Securely signed commits
 * [ ]  Component(s) with PHPDoc blocks, only if necessary or adds value
 * [ ]  Unit testing, with >80% coverage
 * [ ]  User guide updated
 * [x]  Conforms to style guide

### CODE REVIEW

File: system/Debug/Exceptions.php [hunk 1]
```diff
@@ -98,6 +98,9 @@ public function __construct(ExceptionsConfig $config, $request, ResponseInterfac
      * handling of our application.
      *
      * @codeCoverageIgnore
+     *
+     * @return void
+     * @phpstan-return never|void
```
Comment (kenjis) [inline system/Debug/Exceptions.php:103]: It seems `never` is not returned. See https://phpstan.org/writing-php-code/phpdoc-types#bottom-type
```suggestion
```

File: system/Debug/Toolbar.php [hunk 2]
```diff
@@ -348,6 +348,9 @@ protected function roundTo(float $number, int $increments = 5): float
      *
      * @param RequestInterface  $request
      * @param ResponseInterface $response
+     *
+     * @return void
+     * @phpstan-return never|void
```
Comment (kenjis) [inline system/Debug/Toolbar.php:353]: Is there any `exit()` or `die()` in this method?

Comment (kai890707) [reply system/Debug/Toolbar.php]: There does not include exit() or die().
So, I changed there to `@phpstan-return void`.

File: system/Debug/Exceptions.php (hunk 1 above)
Comment (kai890707) [reply system/Debug/Exceptions.php]: I changed there to `@phpstan-return void` .

Comment (kenjis) [reply system/Debug/Exceptions.php]: If `@return` and `@phpstan-return` are the same, `@phpstan-return` is not needed.
Remove it.

Comment (kai890707) [reply system/Debug/Exceptions.php]: done.

File: system/Debug/Toolbar.php (hunk 2 above)
Comment (kai890707) [reply system/Debug/Toolbar.php]: done.

**Review by kenjis — APPROVED** [review]
LGTM!

**Pull request merged**